### PR TITLE
Certificate revocation documentation

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Adds tutorial of how to revoke certificate

**Related issue(s)**

#2314

### CODE REVIEW

File: docs/application-connector/docs/08-02-get-client-certificate.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ After you create an Application (App), connect it to an external solution to con
 
 This guide shows you how to get the client certificate.
 
->**NOTE:** The client certificate is valid for 92 days. See [this](#tutorials-renew-the-client-certificate) tutorial to learn how to renew the client certificate.
+>**NOTE:** The client certificate is valid for 92 days. See [this](#tutorials-renew-the-client-certificate) tutorial to learn how to renew the client certificate. 
+You can also revoke the client certificate, making it unable to be renewed. See [this](#tutorials-revoke-the-certificate) tutorial to learn how to do this.
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-02-get-client-certificate.md]:
```suggestion
You can also revoke the client certificate, which prevents it from being renewed. See [this](#tutorials-revoke-the-client-certificate) tutorial to learn how to do this.
```

File: docs/application-connector/docs/08-08-renew-client-cert.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ type: Tutorials
 
 By default, the client certificate you generate when you connect an external solution to Kyma is valid for 92 days. Follow this tutorial to renew the client certificate.
 
->**NOTE:** You can only renew client certificates that are still valid. If your client certificate expired, you must generate a new one.
+>**NOTE:** You can only renew client certificates that are still valid. If your client certificate is expired or has been revoked, you must generate a new one.
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-08-renew-client-cert.md]:
```suggestion
>**NOTE:** You can only renew client certificates that are still valid. If your client certificate is expired or revoked, you must generate a new one.
```

File: docs/application-connector/docs/08-08-renew-client-cert.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ By default, the client certificate you generate when you connect an external sol
   openssl req -new -sha256 -out renewal.csr -key {PATH_TO_KEY} -subj "{SUBJECT}"
   ```
 
-3. Send a request to the Connector Service to renew the certificate:
+3. To renew a certificate, send a request to RenewCertURL obtained from management/info endpoint. Example call:
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-08-renew-client-cert.md]:
```suggestion
Send a request to the Connector Service to renew the certificate:
```

File: docs/application-connector/docs/08-09-revoke-cert.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Revoke the certificate
+type: Tutorials
+---
+
+This tutorial will show you how to revoke certificate, making it unable to be renewed.
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-09-revoke-cert.md]:
```suggestion
You can revoke a client certificate generated for your Application. Revocation prevents a certificate from being renewed. A revoked certificate, however, continues to be valid until it expires.
```

File: docs/application-connector/docs/08-09-revoke-cert.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: Revoke the certificate
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-09-revoke-cert.md]:
```suggestion
title: Revoke the client certificate
```

File: docs/application-connector/docs/08-09-revoke-cert.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+type: Tutorials
+---
+
+This tutorial will show you how to revoke certificate, making it unable to be renewed.
+
+To revoke certificate make a call to ``certificates/revocations`` endpoint:
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-09-revoke-cert.md]:
```suggestion
To revoke a client certificate, send a request to the `certificates/revocations` endpoint. Pass the certificate you want to revoke and a key that matches this certificate in the call. Run:
```

File: docs/application-connector/docs/08-09-revoke-cert.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    
+```bash
+curl -X POST https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/revocations --cert {CERT_TO_REVOKE} --key {CERT_TO_REVOKE_KEY} -k 
+```
+    
+After successful call, the certificate is revoked and cannot be renewed.
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-09-revoke-cert.md]: Remove this line, brings no value.

File: docs/application-connector/docs/08-09-revoke-cert.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+If you have access to Kyma cluster, you can also revoke a certificate by making a call with certificates sha256 sum to internal ``certificates/revocations`` endpoint:
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/08-09-revoke-cert.md]:
```suggestion
If you have access to the Kyma cluster, you can also revoke a certificate by sending the SHA256 sum of the certificate to the internal `certificates/revocations` endpoint. Run:
```

File: docs/application-connector/docs/08-09-revoke-cert.md (hunk 7 above)
Comment (Maladie) [reply docs/application-connector/docs/08-09-revoke-cert.md]: Done

**Pull request merged**